files-community/Files
# Code Quality: Removed Vanara from DriveHelpers

### Resolved / Related Issues

- https://github.com/files-community/Files/issues/15000

### Steps used to test these changes

1. Set a long drive label
2. See it in Files properly shown
